components/product-details/index.js

```diff
@@ -1,53 +1,67 @@
 import NavBar from '../NavBar';
 import personData from '../../mock/person.json';
 import PropTypes from 'prop-types';
-import path from 'path';
 import Image from 'next/image';
+import Link from 'next/link';
+import { Footer } from '../../components/footer';
+import styles from './product-details.module.css';
+
+const renderUses = (productJSON) =>
+  productJSON.usage.map((use, id) => <p key={id}> {use} </p>);
+
+const renderThumbImage = (productJSON) => {
+  const productArr = [];
+  for (let i = 0; i < 3; i++) {
+    productArr.push({
+      imageUrl: `/assets/${productJSON.image}`,
+      imageId: `${productJSON.id}`,
+    });
+  }
+  return productArr;
+};
```

Comment:
We will have multiple products right? this should be dynamic
Why are we limiting this to only 3?

Reply:
I had no design. So, I just kept 3 for visual purpose.